src/Lib/CachedChannelFactory.cs

```diff
@@ -30,6 +30,27 @@ internal class CachedChannelFactory
         /// </summary>
         private static Dictionary<string, Channel> cache = new Dictionary<string, Channel>();
 
+        /// <summary>
+        /// The grpc setting name for maximum message length that can be received.
+        /// </summary>
+        private const string GRPC_MAX_RECEIVE_MESSAGE_LENGTH_SETTING =
+            "grpc.max_receive_message_length";
+
+        /// <summary>
+        /// The maximum message length that the client library can receive (64 MB).
```

Comment:
Can this say 'receive in bytes'? Just so someone reading the documentation doesn't need to look at the math below to determine it, that would be helpful.

Another option could be changing the variable to be _IN_BYTES.

Reply:
Done